### PR TITLE
New Schedule Meetings are Saved upon closing of application

### PR DESCRIPTION
**Pull request by Brendan8899**
schedule.json is the file that will be updated :)

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -15,19 +15,9 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.ReadOnlyUserPrefs;
-import seedu.address.model.UserPrefs;
+import seedu.address.model.*;
```
Comment (LimKaiWei) [inline src/main/java/seedu/address/MainApp.java:18]: are we allowed to use wildcard for imports?

File: src/main/java/seedu/address/logic/commands/SeeScheduleCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    
+    private final LocalDate date;
+    
+    public SeeScheduleCommand(LocalDate date) {
+        this.date = date;
+    }
+    
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        return null;
```
Comment (LimKaiWei) [inline src/main/java/seedu/address/logic/commands/SeeScheduleCommand.java:25]: i assume this has yet to be implemented and is currently a placeholder for future implementation?

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 3]
```diff
@@ -80,6 +71,9 @@ public Command parseCommand(String userInput) throws ParseException {
 
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
+            
+        case SeeScheduleCommand.COMMAND_WORD:
+
```
Comment (LimKaiWei) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:76]: i assume the same for this?

File: src/main/java/seedu/address/model/person/exceptions/MeetingNotFoundException.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.person.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified meeting.
+ */
+public class MeetingNotFoundException extends RuntimeException {}
```
Comment (LimKaiWei) [inline src/main/java/seedu/address/model/person/exceptions/MeetingNotFoundException.java:6]: should there constructor with a warning message similar to the other exceptions?
```suggestion
public class MeetingNotFoundException extends RuntimeException {
    public MeetingNotFoundException() {
        super("meeting not found");
    }
}
```

**Review by LimKaiWei — COMMENTED** [review]
would you tag this as v1.2 or v1.3 milestone?
remember to create issue to tag this PR and assign yourself to it...

File: schedule.json
Comment (tata32000) [inline schedule.json:1]: Is this for the storage? If so, should this be under `/data` directory?

File: src/main/java/seedu/address/logic/commands/AddScheduleCommand.java [hunk 5]
```diff
@@ -2,11 +2,12 @@
 
 import java.time.LocalDate;
 import java.time.LocalTime;
-import java.util.List;
+import java.util.*;
```
Comment (tata32000) [inline src/main/java/seedu/address/logic/commands/AddScheduleCommand.java:5]: We are not allow to use wildcard imports right?

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 6]
```diff
@@ -23,12 +23,10 @@
 import seedu.address.logic.commands.ListCommand;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.UserPrefs;
+import seedu.address.model.*;
```
Comment (tata32000) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:26]: Same here for the wildcard imports

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 7]
```diff
@@ -18,11 +18,9 @@
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.AddressBook;
-import seedu.address.model.Model;
-import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.*;
```
Comment (tata32000) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:21]: Here also

**Review by tata32000 — CHANGES_REQUESTED** [review]
Maybe need to fix the styling first, the rest looks good

File: src/main/java/seedu/address/MainApp.java (hunk 1 above)
Comment (Brendan8899) [reply src/main/java/seedu/address/MainApp.java:18]: No we are not allowed

File: src/main/java/seedu/address/logic/commands/AddScheduleCommand.java (hunk 5 above)
Comment (Brendan8899) [reply src/main/java/seedu/address/logic/commands/AddScheduleCommand.java:5]: No we are not, my bad, it was auto added

File: src/main/java/seedu/address/logic/commands/SeeScheduleCommand.java (hunk 2 above)
Comment (Brendan8899) [reply src/main/java/seedu/address/logic/commands/SeeScheduleCommand.java:25]: Yes you are right!

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 3 above)
Comment (Brendan8899) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:76]: My bad, no wildcards allowed, autocorrect keeps doing this

File: src/test/java/seedu/address/logic/LogicManagerTest.java (hunk 6 above)
Comment (Brendan8899) [reply src/test/java/seedu/address/logic/LogicManagerTest.java:26]: Yes yes you are correct

**Pull request merged**